sphere-models/src/it/java/io/sphere/sdk/products/attributes/LocalizedToStringProductAttributeConverterTest.java

```diff
@@ -0,0 +1,222 @@
+    }
+
+    @Test
+    public void convertLongSet() {
+        assertThat(converting(ATTR_NAME_NUMBER_SET)).isEqualTo("5, 10");
+    }
+
+    @Test
+    public void convertLong() {
+        assertThat(converting(ATTR_NAME_NUMBER)).isEqualTo("5");
+    }
+
+    @Test
+    public void convertTimeSet() {
+        assertThat(converting(ATTR_NAME_TIME_SET)).isEqualTo("22:05, 23:06");
+    }
+
+    @Test
+    public void convertTime() {
+        assertThat(converting(ATTR_NAME_TIME)).isEqualTo("22:05");
+    }
+
+    @Test
+    public void convertLocalizedStringSet() {
+        assertThat(converting(ATTR_NAME_LOC_TEXT_SET)).isEqualTo("");
+    }
+
+    @Test
+    public void convertLocalizedEnumValueSet() {
+        assertThat(converting(ATTR_NAME_LOC_ENUM_SET)).isEqualTo("drei, zwei");
+    }
+
+    @Ignore
+    @Test
+    public void convertIntegerSet() {
+    }
+
+    @Ignore
+    @Test
+    public void convertInteger() {
+    }
+
+    @Test
+    public void convertEnumValueSet() {
+        assertThat(converting(ATTR_NAME_ENUM_SET)).isEqualTo("three, two");
+    }
+
+    @Ignore
+    @Test
+    public void convertDoubleSet() {
+    }
+
+    @Ignore
+    @Test
+    public void convertDouble() {
+    }
+
+    @Test
+    public void convertDateTimeSet() {
+        assertThat(converting(ATTR_NAME_DATE_TIME_SET)).isEqualTo("11.09.01 22:05, 12.10.02 23:06");
+    }
+
+    @Test
+    public void convertDateSet() {
+        assertThat(converting(ATTR_NAME_DATE_SET)).isEqualTo("11.09.2001, 12.10.2002");
+    }
+
+    @Ignore
+    @Test
+    public void convertChannelReferenceSet() {
+    }
+
+    @Ignore
+    @Test
+    public void convertChannelReference() {
+    }
+
+    @Ignore
+    @Test
+    public void convertCategoryReferenceSet() {
+    }
+
+    @Ignore
+    @Test
+    public void convertCategoryReference() {
+    }
+
+    @Test
+    public void convertBooleanSet() {
+        assertThat(converting(ATTR_NAME_BOOLEAN_SET)).isEqualTo("false, true");
+    }
+
+    @Test
+    public void convertString() {
+        assertThat(converting(ATTR_NAME_TEXT)).isEqualTo("foo");
+    }
+
+    @Test
+    public void convertMoney() {
+        assertThat(converting(ATTR_NAME_MONEY)).isEqualTo("500,00 EUR");
+    }
+
+    @Test
+    public void convertLocalizedString() {
+        assertThat(converting(ATTR_NAME_LOC_TEXT2)).isEqualTo("German foo");
+    }
+
+    @Test
+    public void convertLocalizedEnumValue() {
+        assertThat(converting(ATTR_NAME_LOC_ENUM)).isEqualTo("zwei");
+    }
+
+    @Test
+    public void convertEnumValue() {
+        assertThat(converting(ATTR_NAME_ENUM)).isEqualTo("two");
+    }
+
+    @Test
+    public void convertDateTime() {
+        assertThat(converting(ATTR_NAME_DATE_TIME)).isEqualTo("11.09.01 22:05");
+    }
+
+    @Test
+    public void convertDate() {
+        assertThat(converting(ATTR_NAME_DATE)).isEqualTo("11.09.2001");
+    }
+
+    @Test
+    public void convertBoolean() {
```

Comment:
Maybe I'm not finding the order, but I feel all methods are unsorted, and it makes it difficult to see if we are missing some attribute :P